source/content/addons/object-cache/howto/wordpress.md

```diff
@@ -295,7 +291,55 @@ Refer to the [official Object Cache Pro documentation](https://objectcache.pro/d
 
 	- If you are using WordPress Multisite, subsites do not get their own configuration or graphs. Navigate to `/wp-admin/network/settings.php?page=objectcache` to view network-wide configuration and graphs. This is the only screen throughout the network that displays this information.
 
-### Additional Considerations
+## Local configuration with Lando
```

Comment:
@jazzsequence did you do all these steps yourself? If not, one of us should do them on our machines to confirm accuracy. If you did, I'll mark the PR as approved.

Reply:
yes, but I can check the redis config in the lando.yml one more time